core/kits/manager.php

```diff
@@ -119,6 +121,22 @@ public function create( array $kit_data = [], array $kit_meta_data = [] ) {
 		return $kit->get_id();
 	}
 
+	public function create_new_kit( $kit_name = '', $settings = [], $active = true ) {
+		$kit_name = $kit_name ? $kit_name : esc_html__( 'Custom', 'elementor' );
+
+		$id = $this->create( [
+			'post_title' => $kit_name,
+			'settings' => $settings,
+		] );
+
+		if ( $active ) {
+			update_option( self::OPTION_PREVIOUS, $this->get_active_id() );
+			update_option( self::OPTION_ACTIVE, $id );
+		}
+
+		return $id;
+	}
+
```

Comment:
Maybe add a function for getting the previous id? 
```suggestion
		public function get_previous_id() {
		return get_option( self::OPTION_PREVIOUS);
	}
```

Reply:
Done